nat/nat.go

```diff
@@ -88,7 +88,10 @@ func (p Port) Int() int {
 
 // Range returns the start/end port numbers of a Port range as ints
 func (p Port) Range() (int, int, error) {
-	return ParsePortRangeToInt(p.Port())
+	// We don't need to check for an error because we're going to
+	// assume that any error would have been found, and reported, in NewPort()
+	start, end, _ := ParsePortRangeToInt(p.Port())
+	return start, end, nil
```

Comment:
I'm a bit on the fence here; the `Int()` function does not have an error-return in its signature, so even if the error would _not_ be ignored, there's nowhere to send it to (other than logging or a panic); https://github.com/docker/go-connections/blob/fa09c952e3eadbffaf8afc5b8a1667158ba38ace/nat/nat.go#L80-L87


The `Range()` function _does_ have an error return, so the error can be handled (or ignored) by consumers.

Was there an issue you ran into, other than error-handling being inconsistent?


(for transparency; the code in this repository is quite old, and not all parts are very well designed; we're still evaluating the future of this repository, and possibly breaking it up  / moving code elsewhere)

Reply:
No, inconsistent error handling was the only issue.

Okay, if there is a possibility that the code is being moved or broken up, this PR won't contribute much. I guess we can just close it.